store/tikv/coprocessor.go

```diff
@@ -691,9 +692,10 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *Backoffer, task *copTask, ch
 // meet the secondary lock again and run into a deadloop.
 type clientHelper struct {
 	*LockResolver
-	*RegionRequestSender
+	*RegionCache
+	Client
 
-	minCommitTSPushed []uint64
+	minCommitTSPushed map[uint64]struct{}
```

Comment:
We can add a slice along with the map, so we don't need to make a slice for every request.

Reply:
Done